config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  use_doorkeeper scope: 'oauth' do
+    controllers applications: 'oauth/applications'
```

Comment:
oops, I ended up not needing to subclass the controller, but missed that I left this code in.  (I have some wip in another branch)